packages/vite/src/node/plugins/html.ts

```diff
@@ -317,11 +320,20 @@ function handleParseError(
       // lit generates <?>: https://github.com/lit/lit/issues/2470
       return
   }
+  fileErrorInfo[filePath] ??= {}
+  if (fileErrorInfo[filePath][parserError.code]) {
+    // already warned about this error
+    return
+  }
+  fileErrorInfo[filePath][parserError.code] = true
   const parseError = formatParseError(parserError, filePath, html)
-  throw new Error(
-    `Unable to parse HTML; ${parseError.message}\n` +
-      ` at ${parseError.loc.file}:${parseError.loc.line}:${parseError.loc.column}\n` +
-      `${parseError.frame}`,
+
+  warn(
+    colors.yellow(
+      `Unable to parse HTML; ${parseError.message}\n` +
+        ` at ${parseError.loc.file}:${parseError.loc.line}:${parseError.loc.column}\n` +
+        `${parseError.frame.length > 300 ? '[this line is omitted as the content was too long] ' : parseError.frame}`,
```

Comment:
Since we are omitting the whole code frame instead of a specific line.
```suggestion
        `${parseError.frame.length > 300 ? '[this code frame is omitted as the content was too long] ' : parseError.frame}`,
```